albumentations/core/transforms_interface.py

```diff
@@ -105,29 +114,29 @@ def apply_with_params(self, params, force_apply=False, **kwargs):  # skipcq: PYL
                 res[key] = None
         return res
 
-    def set_deterministic(self, flag, save_key="replay"):
+    def set_deterministic(self, flag: bool, save_key: str = "replay") -> BasicTransform:
```

Comment:
Remove `from __future__ import annotations` and change this line to `def set_deterministic(self, flag: bool, save_key: str = "replay") -> "BasicTransform":`